src/app/submissions/submissions.component.html

```diff
@@ -52,7 +52,7 @@
         </mat-cell>
       </ng-container>
       <mat-header-row *matHeaderRowDef="displayedColumns" class="hide"></mat-header-row>
-      <mat-row *matRowDef="let row; columns: displayedColumns;" (click)="submissionAction(row)"></mat-row>
+      <mat-row *matRowDef="let row; columns: displayedColumns;" (click)="submissionAction(row)" [ngClass]="{'mat-row-pointer': row.status=='complete'}"></mat-row>
```

Comment:
A few notes:
* There is a global `cursor-pointer` class that would be better in this case.
* We want there to be a pointer if the submissions status does not equal pending, not only when it equals complete.  There are three states, so this will remove the pointer for the third as well.
* Please always use `===` or `!==` instead of `==` or `!=`